app/jobs/resolution_proofing_job.rb

```diff
@@ -62,18 +62,26 @@ def proof_lexisnexis_then_aamva(timer:, applicant_pii:, should_proof_state_id:)
     result = proofer_result.to_h
     resolution_success = proofer_result.success?
 
+    result[:transaction_id] = proofer_result.transaction_id
+
+    exception = proofer_result.exception.inspect if proofer_result.exception
+    result[:timed_out] = proofer_result.timed_out?
+    result[:exception] = exception
+
     result[:context] = {
-      stages: [
-        {
-          resolution: resolution_proofer.class.vendor_name,
+      dob_year_only: false,
```

Comment:
logging `dob_year_only` and `should_proof_state_id` gives us the opportunity to see success rates for different combinations